database/download.go

```diff
@@ -2,11 +2,20 @@ package database
 
 import (
 	"context"
+	"fmt"
 	"time"
 
 	"gitlab.com/NebulousLabs/errors"
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+const (
+	// DownloadUpdateWindow defines a time window during which instead of
+	// creating a new download record for the given skylink, we'll update the
+	// previous one, as long as it has been updated within the window.
+	DownloadUpdateWindow = 10 * time.Minute
```

Comment:
Yes, this is very arbitrary. I'm very much open to changing this to anything that makes more sense. 

I chose the period because it fits the use case of watching a movie and stopping for a quick bio break or to get a snack.